src/threads.cpp

```diff
@@ -297,30 +231,19 @@ void MapperClass::OctomappingTask() {
         // Get Point Cloud
         mutexes_.point_cloud.lock();
             // Get data from queue
-            pcl::PointCloud< pcl::PointXYZ > point_cloud = globals_.pcl_queue.front().cloud;
-            const tf::StampedTransform tf_cam2world = globals_.pcl_queue.front().tf_cam2world;
+            pcl::PointCloud< pcl::PointXYZ > pcl_sensor_frame = globals_.pcl_queue.front().cloud;
+            tf_pcl2world = globals_.pcl_queue.front().tf_pcl2world;
             const bool is_lidar = globals_.pcl_queue.front().is_lidar;
 
             // Remove data from queue
             globals_.pcl_queue.pop();
         mutexes_.point_cloud.unlock();
 
-        // Check if a tf message has been received already. If not, return
-        if (tf_cam2world.stamp_.toSec() == 0) {
-            continue;
-        }
-
-        // Get camera transform
-        tf::Quaternion q = tf_cam2world.getRotation();
-        tf::Vector3 v = tf_cam2world.getOrigin();
-        Eigen::Affine3d transform = Eigen::Affine3d::Identity();
-        transform.translation() << v.getX(), v.getY(), v.getZ();
-        transform.rotate(Eigen::Quaterniond(q.getW(), q.getX(), q.getY(), q.getZ()));
```

Comment:
This portion of the code was sent to `msg_conversions::tf_transform_to_eigen_transform`